RefactorThis.Application/ProductOptions/CreateProductOptionCommand/CreateProductOptionCommandHandler.cs

```diff
@@ -1,11 +1,21 @@
+using Microsoft.Data.Sqlite;
 using RefactorThis.Application.Abstractions;
+using RefactorThis.Domain.Entities;
 
 namespace RefactorThis.Application.ProductOptions.CreateProductOptionCommand;
 
-public class CreateProductOptionCommandHandler : ICommandHandler<CreateProductOptionCommand, Guid>
+public class CreateProductOptionCommandHandler : ICommandHandler<CreateProductOptionCommand>
 {
-    public Task<Guid> Handle(CreateProductOptionCommand request, CancellationToken cancellationToken)
+    public Task Handle(CreateProductOptionCommand request, CancellationToken cancellationToken)
     {
-        throw new NotImplementedException();
+        var conn = Helpers.NewConnection<SqliteConnection>();
```

Comment:
for this line instead of using Helpers, create a new interface `ISqlDataConnectionFactory` that have one method `CreateConnection()` and implementation should accept constructor injection `IOptions<DatabaseConfig> databaseConfig`

```public class DatabaseConfig {

public string ConnectionString { get; set }

}```